addons/uom/models/uom_uom.py

```diff
@@ -41,7 +41,7 @@ class UoM(models.Model):
 
     _sql_constraints = [
         ('factor_gt_zero', 'CHECK (factor!=0)', 'The conversion ratio for a unit of measure cannot be 0!'),
-        ('rounding_gt_zero', 'CHECK (rounding>0)', 'The rounding precision must be greater than 0!')
+        ('rounding_gt_zero', 'CHECK (rounding!=0)', 'The rounding precision must be strictly positive.')
```

Comment:
i don't get it
0.000 should not pass the original "rounding_gt_zero" constraint

Reply:
@nsa-odoo No need to change this it will work what as it is ( rounding > 0 ) just change message ...

Reply:
actually the original feedback is there because we're able to set a rounding of 0.000 
can you double check this usecase?

Reply:
actually the original feedback is there because we're able to set a rounding of 0.000 
can you double check this usecase?

Reply:
@sle-odoo When you set rounding 0.000 , it will raise warning but when you close the warning **view not reloaded**  and its not going to change a value in cache, 
And default value of rounding is 0.01 , so system not giving warning second time...

Reply:
@sle-odoo 
it's not producible everytime...
https://drive.google.com/open?id=1R8LC2PWmuRPjKzWIdnbWLhvrt3AmqZAj

Reply:
@sle-odoo 
https://drive.google.com/open?id=1hWZ891OU5pSpKtHJwXpT5d9Fi4o5qZ7q